scripts/pelorus-operator-patches/07_spec_description.diff

```diff
@@ -0,0 +1,89 @@
+--- config/crd/bases/charts.pelorus.konveyor.io_pelorus.yaml.original	2022-12-16 11:41:22.683297131 +0100
++++ config/crd/bases/charts.pelorus.konveyor.io_pelorus.yaml	2022-12-16 11:40:22.725983634 +0100
+@@ -15,29 +15,76 @@
+   - name: v1alpha1
+     schema:
+       openAPIV3Schema:
+-        description: Pelorus is the Schema for the pelorus API
++        description: >-
```

Comment:
I do not know how the editing of the diff files works, but I would change the `>-`s for `|`s, for breaking lines in yaml, just for standardization

Reply:
That is what OpenShift editor gives and I prefer to use ">-" as it's easier to copy-paste from the OCP editor.

Reply:
alright, no problem